themes-default/slim/static/js/api.js

```diff
@@ -1,6 +1,34 @@
 const baseUrl = $('body').attr('api-root');
 const idToken = $('body').attr('api-key');
 
+/**
+ * Api client based on the axios client, to communicate with medusa's web routes, which return json data.
+ */
+const apiRoute = axios.create({ // eslint-disable-line no-unused-vars
+    baseURL: baseUrl.replace('api/v2/', ''),
+    timeout: 10000,
+    headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        'X-Api-Key': idToken
+    }
+});
+
+/**
+ * Api client based on the axios client, to communicate with medusa's api v1.
+ */
+const apiv1 = axios.create({ // eslint-disable-line no-unused-vars
+    baseURL: baseUrl.replace('api/v2/', 'api/v1/' + idToken + '/'),
```

Comment:
Any reason not to use the header like the other?

Reply:
I thought the api v1 does not use it?
And I checked apibuilder, and there it's adding the token to the query string?

Reply:
Ignore this as well. I got this and other other section mixed up.